projects/lib-shared-modules/src/lib/components/card/card.component.scss

```diff
@@ -18,15 +20,23 @@
     justify-content: space-between;
 }
 .chip-container{
-    background-color: $chipPurple;  
-    border-radius: 25px;
+    background-color: $chipPurple;
+    border-radius: 20px;
     padding: 0 10px;
 }
 .text{
     color: $textBlue;
     margin: 0;
     padding: 0;
-    
+}
+.overflow-ellipsis{
+    overflow: hidden;
+    text-overflow:ellipsis;
+    white-space: nowrap;
+    white-space: pre-line;
+    display: -webkit-box;
+    -webkit-line-clamp: 1;
```

Comment:
why we need this ?

Reply:
it will handle 1 line and width according to card size